Products/SQLAlchemyDA/da.py

```diff
@@ -65,12 +103,13 @@ class SAWrapper(SimpleItem, PropertyManager):
     quoting_style = 'standard'
     _isAnSQLConnection = True
     extra_engine_options = ()
-
+    
     security = ClassSecurityInfo()
 
     def __init__(self, id, title=''):
         self.id = id
         self.title = title
+        register_da(self.id, self)
```

Comment:
How do you deal with name clashes?  E.g., what happens if somebody defines DAs in two different folders, but calls them both `da`?